### PR TITLE
Converted the route as mentioned

### LINKED ISSUE #563: Enable code splitting at the route level
### Feature Summary

Async load top level view components to take advantage of code splitting

### Detailed Description

Convert routes to something like this:
```
const routes = [
    {
        path: '/path',
        component: () => import('/path/to/component')
    }
]
```

### PR DESCRIPTION
**Pull request by ar1701**
## Issue number- #563 

Relevant [#563 ](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- Resolves #563 : Converted the Route 

## Please check the following

- [x] Do the tests still pass? (see [Run the Tests](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#run-the-tests))
- [x] Is the code formatted properly? (see [Linting (Formatting)](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#linting-formatting))
- For New Features:
  - [x] Have tests been added to cover any new features or fixes?
  - [x] Has the documentation been updated accordingly?

### CODE REVIEW

File: src/router.js [hunk 1]
```diff
@@ -63,28 +57,26 @@ const checkAndSubscribeToLobby = async (to) => {
...
   {
     path: '/signup',
     name: ROUTE_NAME_SIGNUP,
-    component: LoginView,
+    component: () => import('@/routes/login/LoginView.vue'),
     meta: {
       hideNavigation: true,
     },
   },
-  // This route is just a passthrough to make sure the user is fully logged out before putting
```
Comment (seriouslysean) [inline src/router.js:86]: I’m going to assume you used AI a bit to help with this task as it tends to remove comments unless you specify that the model keep them, and that’s totally fine, but we need these comments to be here.

**Review by seriouslysean — CHANGES_REQUESTED** [review]
Great first attempt! Rename the title of your PR to better reflect the goal (implement route splitting), and add those comments back.

Thanks for contributing!

File: src/router.js (hunk 1 above)
Comment (ar1701) [reply src/router.js:86]: Yea, little bit. Just for refactoring and little bit for the understanding about the code

**Review by ar1701 — COMMENTED** [review]
I took the help of ai for the of get more undestanding about the code.

**Comment by seriouslysean** [comment]
@ar1701 I love AI and use it all the time! So helpful to learn and it can offer a lot of insight (and power, haha).

That said, it’s incredibly important to double check what it’s telling you and also what it’s doing. When learning, I recommend asking it what you need to do, and then making the changes yourself until you get more comfortable.

File: src/router.js (hunk 1 above)
Comment (itsalaidbacklife) [reply src/router.js:86]: @ar1701 if you would still like to complete this issue I believe all that remains is re-adding the previously deleted comments

Comment (ar1701) [reply src/router.js:86]: Ok...i will do it by tonight

Comment (ar1701) [reply src/router.js:86]: @itsalaidbacklife I have added the previous comments and did some modifications . Please verify once

**Comment by itsalaidbacklife** [comment]
@ar1701 just so you know for next time, you don't need to open a new PR to make changes to an existing one. You can simply commit and push the changes to your branch and the PR will be updated to reflect those changes. 

That's because a pull request is just a UI for merging one branch into another. So whenever we approve and merge a PR, it merges the current latest state of the feature branch and merges that into the base branch (main).

For now, I'm closing this as a duplicate of #1100 since that has the latest changes

File: src/router.js (hunk 1 above)
Comment (seriouslysean) [reply src/router.js:86]: Just make sure you push it up, I don’t see those changes yet. Looking forward to it!

**Pull request closed without merging**